features/step_definitions/owner.steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I visit the owner site$/) do
+  visit '/owner'
+end
+
+When(/^owner adds dish$/) do
+  visit '/owner'
+  fill_in 'name', :with=> 'dish'
+  fill_in 'price', :with=> '34'
+end
```

Comment:
Are you testing these methods somewhere else in the site? If not, they should each have their own line here. The test would look like:
```
Given I visit the owner site
And I fill in "name" with "dish"
And I fill in "price" with "34"
And I click "Submit"
Then I should see "Whatever"
```